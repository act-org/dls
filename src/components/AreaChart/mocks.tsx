/**
 * Copyright (c) Encoura, LLC and its affiliates.
 *
 * This source code is licensed under the MIT license found in the
 * LICENSE file in the root directory of this source tree.
 */

export const percentageData = [
  {
    A: 0.2,
    name: 'Day 1',
  },
  {
    A: 0.6,
    name: 'Day 2',
  },
  {
    A: 0.35,
    name: 'Day 3',
  },
  {
    A: 0.34,
    name: 'Day 4',
  },
  {
    A: 0.82,
    name: 'Day 5',
  },
  {
    A: 0.2,
    name: 'Day 6',
  },
  {
    A: 1,
    name: 'Day 7',
  },
];

export const defaultData = [
  {
    A: 7,
    name: 'Day 1',
  },
  {
    A: 15,
    name: 'Day 2',
  },
  {
    A: 18,
    name: 'Day 3',
  },
  {
    A: 12,
    name: 'Day 4',
  },
  {
    A: 4,
    name: 'Day 5',
  },
  {
    A: 9,
    name: 'Day 6',
  },
  {
    A: 11,
    name: 'Day 7',
  },
  {
    A: 4,
    name: 'Day 7',
  },
];

export const largerDataset = [
  {
    A: 2,
    B: 8,
    name: 'Day 1',
  },
  {
    A: 5,
    B: 17,
    name: 'Day 2',
  },
  {
    A: 18,
    B: 18,
    name: 'Day 3',
  },
  {
    A: 4,
    B: 24,
    name: 'Day 4',
  },
  {
    A: 22,
    B: 6,
    name: 'Day 5',
  },
  {
    A: 15,
    B: 4,
    name: 'Day 6',
  },
];

export const largerDatasetWith18Keys: { [key: string]: number | string }[] =
  largerDataset.map(item => {
    const newItem: { [key: string]: number | string } = { name: item.name };
    // eslint-disable-next-line no-loops/no-loops
    for (let i = 0; i < 18; i += 1) {
      newItem[String.fromCharCode(65 + i)] = Math.floor(Math.random() * 100);
    }
    return newItem;
  });
