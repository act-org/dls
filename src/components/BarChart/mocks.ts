export const defaultData = [
  {
    '2019': 138598,
    '2019scatter': 60000,
    '2020': 130127,
    '2020scatter': 30000,
    '2021': 9250,
    '2021scatter': 40000,
    name: 'California',
  },
  {
    '2019': 57500,
    '2019scatter': 4150,
    '2020': 56500,
    '2020scatter': 5600,
    '2021': 58000,
    '2021scatter': 8000,
    name: 'Nevada',
  },
  {
    '2019': 44800,
    '2019scatter': 7410,
    '2020': 43800,
    '2020scatter': 3250,
    '2021': 45000,
    '2021scatter': 4000,
    name: 'New York',
  },
  {
    '2019': 13000,
    '2019scatter': 900,
    '2020': 15000,
    '2020scatter': 800,
    '2021': 14000,
    '2021scatter': 1900,
    name: 'Texas',
  },
  {
    '2019': 12000,
    '2019scatter': 1600,
    '2020': 15500,
    '2020scatter': 2000,
    '2021': 15000,
    '2021scatter': 750,
    name: 'Washington',
  },
  {
    '2019': 0,
    '2019scatter': 1200,
    '2020': 9800,
    '2020scatter': 1450,
    '2021': 12500,
    '2021scatter': 1600,
    name: 'Oregon',
  },
];

export const defaultDataFourLines = defaultData.map(d => ({ ...d, '2022': 7000, '2022scatter': 3500 }));

export const longLabelData = [
  {
    '2019': 900,
    '2020': 823,
    '2021': 925,
    name: ' Los Angeles - California, United States',
  },
  {
    '2019': 575,
    '2020': 565,
    '2021': 580,
    name: 'Carson City - Nevada, United States',
  },
  {
    '2019': 448,
    '2020': 438,
    '2021': 450,
    name: 'New York - New York, United States',
  },
];

export const longSubLabelData = [
  {
    '2019 with long sublabel': 900,
    '2020 with long sublabel': 823,
    '2021 with long sublabel': 925,
    name: ' California',
  },
  {
    '2019 with long sublabel': 575,
    '2020 with long sublabel': 565,
    '2021 with long sublabel': 580,
    name: ' Nevada',
  },
  {
    '2019 with long sublabel': 448,
    '2020 with long sublabel': 438,
    '2021 with long sublabel': 450,
    name: 'New York',
  },
];

export const defaultBarKeys = ['2019', '2020', '2021'];

export const defaultBarKeysFourLines = [...defaultBarKeys, '2022'];

export const longBarKeys = ['2019 with long sublabel', '2020 with long sublabel', '2021 with long sublabel'];
