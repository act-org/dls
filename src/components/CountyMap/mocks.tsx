/**
 * Copyright (c) Encoura, LLC and its affiliates.
 *
 * This source code is licensed under the MIT license found in the
 * LICENSE file in the root directory of this source tree.
 */

import { range } from 'd3-array';
import { scaleQuantile } from 'd3-scale';

import { IMapDataProps } from '~/types';

interface MappedFeaturesValueProps {
  quantity: number;
  stateCode: string;
}

interface MappedFeaturesProps {
  [key: string]: MappedFeaturesValueProps;
}

export const defaultData = [
  {
    CONVERSION_QUANTITY: 547,
    COUNTY_NAME: 'Nassau',
    FIPS: '36059',
    QUANTITY: 883,
    STATE_CODE: 'NY',
  },
  {
    CONVERSION_QUANTITY: 457,
    COUNTY_NAME: 'Suffolk',
    FIPS: '36103',
    QUANTITY: 761,
    STATE_CODE: 'NY',
  },
  {
    CONVERSION_QUANTITY: 301,
    COUNTY_NAME: 'Westchester',
    FIPS: '36119',
    QUANTITY: 689,
    STATE_CODE: 'NY',
  },
  {
    CONVERSION_QUANTITY: 316,
    COUNTY_NAME: 'Bergen',
    FIPS: '34003',
    QUANTITY: 567,
    STATE_CODE: 'NJ',
  },
  {
    CONVERSION_QUANTITY: 281,
    COUNTY_NAME: 'Capitol',
    FIPS: '09110',
    QUANTITY: 538,
    STATE_CODE: 'CT',
  },
  {
    CONVERSION_QUANTITY: 27,
    COUNTY_NAME: 'Bronx',
    FIPS: '36005',
    QUANTITY: 400,
    STATE_CODE: 'NY',
  },
  {
    CONVERSION_QUANTITY: 72,
    COUNTY_NAME: 'Queens',
    FIPS: '36081',
    QUANTITY: 276,
    STATE_CODE: 'NY',
  },
  {
    CONVERSION_QUANTITY: 112,
    COUNTY_NAME: 'Dutchess',
    FIPS: '36027',
    QUANTITY: 260,
    STATE_CODE: 'NY',
  },
  {
    CONVERSION_QUANTITY: 156,
    COUNTY_NAME: 'Monmouth',
    FIPS: '34025',
    QUANTITY: 258,
    STATE_CODE: 'NJ',
  },
  {
    CONVERSION_QUANTITY: 48,
    COUNTY_NAME: 'Kings',
    FIPS: '36047',
    QUANTITY: 254,
    STATE_CODE: 'NY',
  },
];

export const defaultProcessDataFn = (
  featureCollection: GeoJSON.FeatureCollection<GeoJSON.Geometry>,
  data: Array<IMapDataProps>,
): GeoJSON.FeatureCollection<GeoJSON.Geometry> => {
  const { features } = featureCollection;

  const scale = scaleQuantile()
    .domain(data.map(item => item.QUANTITY as number))
    .range(range(10) as Iterable<number>);

  const mappedFeatures = data.reduce(
    (acc: MappedFeaturesProps, { COUNTY_NAME, QUANTITY, STATE_CODE }) => {
      acc[COUNTY_NAME as string] = {
        quantity: QUANTITY as number,
        stateCode: STATE_CODE as string,
      };
      return acc;
    },
    {},
  );

  return {
    features: features.map(f => {
      if (f.properties?.NAME && mappedFeatures[f.properties.NAME]) {
        const quantity =
          f.properties?.NAME && mappedFeatures[f.properties.NAME]
            ? scale(mappedFeatures[f.properties.NAME].quantity)
            : 0;

        const properties = {
          ...f.properties,
          quantity,
          stateCode: mappedFeatures[f.properties.NAME].stateCode,
          value:
            f.properties?.NAME && mappedFeatures[f.properties.NAME]
              ? mappedFeatures[f.properties.NAME].quantity
              : 0,
        };

        return { ...f, properties };
      }

      return { ...f };
    }),
    type: 'FeatureCollection',
  };
};
