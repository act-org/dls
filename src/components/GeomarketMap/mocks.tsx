/**
 * Copyright (c) ACT, Inc. and its affiliates.
 *
 * This source code is licensed under the MIT license found in the
 * LICENSE file in the root directory of this source tree.
 *
 * @prettier
 */

import { range } from 'd3-array';
import { scaleQuantile } from 'd3-scale';

import { IMapDataProps } from '~/types';

export const defaultData = [
  {
    EPS_CODE: 'MS-01',
    EPS_NAME: 'Jackson',
    SOURCE_COUNT1: 37264,
    SOURCE_COUNT2: 47487,
    SOURCE_COUNT3: 49788,
    STATE_CODE: 'MS',
  },
  {
    EPS_CODE: 'FL-07',
    EPS_NAME: 'Collier, Hendry, & Monroe Co',
    SOURCE_COUNT1: 4382,
    SOURCE_COUNT2: 4687,
    SOURCE_COUNT3: 4226,
    STATE_CODE: 'FL',
  },
  {
    EPS_CODE: 'CA-32',
    EPS_NAME: 'Central Valley--North',
    SOURCE_COUNT1: 19812,
    SOURCE_COUNT2: 18554,
    SOURCE_COUNT3: 13355,
    STATE_CODE: 'CA',
  },
  {
    EPS_CODE: 'OH-03',
    EPS_NAME: 'City of Cleveland (West)',
    SOURCE_COUNT1: 7669,
    SOURCE_COUNT2: 8218,
    SOURCE_COUNT3: 7233,
    STATE_CODE: 'OH',
  },
  {
    EPS_CODE: 'IN-09',
    EPS_NAME: 'South Central Indiana',
    SOURCE_COUNT1: 6930,
    SOURCE_COUNT2: 6633,
    SOURCE_COUNT3: 5265,
    STATE_CODE: 'IN',
  },
  {
    EPS_CODE: 'NJ-09',
    EPS_NAME: 'Hudson Co',
    SOURCE_COUNT1: 5404,
    SOURCE_COUNT2: 5471,
    SOURCE_COUNT3: 4476,
    STATE_CODE: 'NJ',
  },
  {
    EPS_CODE: 'TN-01',
    EPS_NAME: 'Chattanooga',
    SOURCE_COUNT1: 16989,
    SOURCE_COUNT2: 22150,
    SOURCE_COUNT3: 15835,
    STATE_CODE: 'TN',
  },
  {
    EPS_CODE: 'NH-02',
    EPS_NAME: 'Merrimack Valley',
    SOURCE_COUNT1: 3246,
    SOURCE_COUNT2: 3234,
    SOURCE_COUNT3: 2280,
    STATE_CODE: 'NH',
  },
  {
    EPS_CODE: 'AL-02',
    EPS_NAME: 'Huntsville & Florence',
    SOURCE_COUNT1: 20490,
    SOURCE_COUNT2: 26773,
    SOURCE_COUNT3: 27957,
    STATE_CODE: 'AL',
  },
  {
    EPS_CODE: 'TX-17',
    EPS_NAME: 'City of Houston (East)',
    SOURCE_COUNT1: 11452,
    SOURCE_COUNT2: 11990,
    SOURCE_COUNT3: 9168,
    STATE_CODE: 'TX',
  },
  {
    EPS_CODE: 'TX-14',
    EPS_NAME: 'City of San Antonio',
    SOURCE_COUNT1: 7587,
    SOURCE_COUNT2: 9543,
    SOURCE_COUNT3: 8476,
    STATE_CODE: 'TX',
  },
  {
    EPS_CODE: 'PA-10',
    EPS_NAME: 'Southern Pennsylvania (East)',
    SOURCE_COUNT1: 11557,
    SOURCE_COUNT2: 10589,
    SOURCE_COUNT3: 9214,
    STATE_CODE: 'PA',
  },
];

interface MappedFeaturesValueProps {
  quantity: number;
  stateCode: string;
}

interface MappedFeaturesProps {
  [key: string]: MappedFeaturesValueProps;
}

export const defaultProcessDataFn = (
  featureCollection: GeoJSON.FeatureCollection<GeoJSON.Geometry>,
  data: Array<IMapDataProps>,
): GeoJSON.FeatureCollection<GeoJSON.Geometry> => {
  const { features } = featureCollection;

  const scale = scaleQuantile()
    .domain(data.map(item => item.SOURCE_COUNT1 as number))
    .range(range(10) as Iterable<number>);

  const mappedFeatures = data.reduce(
    (acc: MappedFeaturesProps, { SOURCE_COUNT1, EPS_CODE, STATE_CODE }) => {
      acc[EPS_CODE as string] = {
        quantity: SOURCE_COUNT1 as number,
        stateCode: STATE_CODE as string,
      };
      return acc;
    },
    {},
  );

  return {
    features: features.map(f => {
      if (f.id && mappedFeatures[f.id]) {
        const properties = {
          ...f.properties,
          quantity: scale(mappedFeatures[f.id].quantity),
          stateCode: mappedFeatures[f.id].stateCode,
          value: mappedFeatures[f.id].quantity,
        };

        return { ...f, properties };
      }

      return { ...f };
    }),
    type: 'FeatureCollection',
  };
};
