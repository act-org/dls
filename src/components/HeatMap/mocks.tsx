/**
 * Copyright (c) ACT, Inc. and its affiliates.
 *
 * This source code is licensed under the MIT license found in the
 * LICENSE file in the root directory of this source tree.
 */

import type { Point } from 'geojson';

export const defaultData = [
  {
    LATITUDE: 47.588294,
    LONGITUDE: -122.034863,
    SOURCE_COUNT: 420,
    STATE_CODE: 'WA',
    ZIP: '98075',
  },
  {
    LATITUDE: 35.883237,
    LONGITUDE: -86.961133,
    SOURCE_COUNT: 1262,
    STATE_CODE: 'TN',
    ZIP: '37064',
  },
  {
    LATITUDE: 33.778829,
    LONGITUDE: -117.326646,
    SOURCE_COUNT: 368,
    STATE_CODE: 'CA',
    ZIP: '92570',
  },
  {
    LATITUDE: 40.218356,
    LONGITUDE: -84.652582,
    SOURCE_COUNT: 53,
    STATE_CODE: 'OH',
    ZIP: '45303',
  },
  {
    LATITUDE: 41.100602,
    LONGITUDE: -85.180709,
    SOURCE_COUNT: 179,
    STATE_CODE: 'IN',
    ZIP: '46808',
  },
  {
    LATITUDE: 39.796571,
    LONGITUDE: -83.816142,
    SOURCE_COUNT: 2,
    STATE_CODE: 'OH',
    ZIP: '45316',
  },
  {
    LATITUDE: 36.111646,
    LONGITUDE: -115.211941,
    SOURCE_COUNT: 544,
    STATE_CODE: 'NV',
    ZIP: '89103',
  },
  {
    LATITUDE: 33.110098,
    LONGITUDE: -87.552052,
    SOURCE_COUNT: 700,
    STATE_CODE: 'AL',
    ZIP: '35405',
  },
  {
    LATITUDE: 40.771522,
    LONGITUDE: -82.53824,
    SOURCE_COUNT: 436,
    STATE_CODE: 'OH',
    ZIP: '44903',
  },
  {
    LATITUDE: 41.39897,
    LONGITUDE: -83.68613,
    SOURCE_COUNT: 396,
    STATE_CODE: 'OH',
    ZIP: '43402',
  },
  {
    LATITUDE: 34.231248,
    LONGITUDE: -118.245509,
    SOURCE_COUNT: 204,
    STATE_CODE: 'CA',
    ZIP: '91214',
  },
  {
    LATITUDE: 40.692911,
    LONGITUDE: -73.856759,
    SOURCE_COUNT: 133,
    STATE_CODE: 'NY',
    ZIP: '11421',
  },
  {
    LATITUDE: 40.58013,
    LONGITUDE: -83.769524,
    SOURCE_COUNT: 56,
    STATE_CODE: 'OH',
    ZIP: '43310',
  },
  {
    LATITUDE: 34.709737,
    LONGITUDE: -86.742356,
    SOURCE_COUNT: 944,
    STATE_CODE: 'AL',
    ZIP: '35758',
  },
  {
    LATITUDE: 34.767581,
    LONGITUDE: -82.221576,
    SOURCE_COUNT: 818,
    STATE_CODE: 'SC',
    ZIP: '29681',
  },
  {
    LATITUDE: 41.451328,
    LONGITUDE: -81.393304,
    SOURCE_COUNT: 335,
    STATE_CODE: 'OH',
    ZIP: '44022',
  },
  {
    LATITUDE: 29.614261,
    LONGITUDE: -95.391041,
    SOURCE_COUNT: 237,
    STATE_CODE: 'TX',
    ZIP: '77047',
  },
  {
    LATITUDE: 38.83447,
    LONGITUDE: -97.641195,
    SOURCE_COUNT: 580,
    STATE_CODE: 'KS',
    ZIP: '67401',
  },
  {
    LATITUDE: 34.931311,
    LONGITUDE: -78.920537,
    SOURCE_COUNT: 1179,
    STATE_CODE: 'NC',
    ZIP: '28306',
  },
  {
    LATITUDE: 39.895188,
    LONGITUDE: -75.040811,
    SOURCE_COUNT: 149,
    STATE_CODE: 'NJ',
    ZIP: '08033',
  },
  {
    LATITUDE: 26.191556,
    LONGITUDE: -81.804294,
    SOURCE_COUNT: 98,
    STATE_CODE: 'FL',
    ZIP: '34103',
  },
  {
    LATITUDE: 40.725647,
    LONGITUDE: -82.52743,
    SOURCE_COUNT: 171,
    STATE_CODE: 'OH',
    ZIP: '44907',
  },
  {
    LATITUDE: 34.941882,
    LONGITUDE: -92.068454,
    SOURCE_COUNT: 717,
    STATE_CODE: 'AR',
    ZIP: '72023',
  },
  {
    LATITUDE: 34.798828,
    LONGITUDE: -91.907652,
    SOURCE_COUNT: 253,
    STATE_CODE: 'AR',
    ZIP: '72086',
  },
  {
    LATITUDE: 36.641254,
    LONGITUDE: -119.904763,
    SOURCE_COUNT: 251,
    STATE_CODE: 'CA',
    ZIP: '93706',
  },
  {
    LATITUDE: 30.265387,
    LONGITUDE: -91.645057,
    SOURCE_COUNT: 413,
    STATE_CODE: 'LA',
    ZIP: '70582',
  },
  {
    LATITUDE: 44.874438,
    LONGITUDE: -93.334673,
    SOURCE_COUNT: 137,
    STATE_CODE: 'MN',
    ZIP: '55435',
  },
  {
    LATITUDE: 41.504386,
    LONGITUDE: -81.554126,
    SOURCE_COUNT: 481,
    STATE_CODE: 'OH',
    ZIP: '44118',
  },
  {
    LATITUDE: 44.590016,
    LONGITUDE: -91.505874,
    SOURCE_COUNT: 59,
    STATE_CODE: 'WI',
    ZIP: '54738',
  },
  {
    LATITUDE: 39.676683,
    LONGITUDE: -83.902802,
    SOURCE_COUNT: 622,
    STATE_CODE: 'OH',
    ZIP: '45385',
  },
  {
    LATITUDE: 25.917628,
    LONGITUDE: -80.396594,
    SOURCE_COUNT: 483,
    STATE_CODE: 'FL',
    ZIP: '33018',
  },
  {
    LATITUDE: 32.380803,
    LONGITUDE: -90.306106,
    SOURCE_COUNT: 540,
    STATE_CODE: 'MS',
    ZIP: '39209',
  },
  {
    LATITUDE: 30.163983,
    LONGITUDE: -97.822647,
    SOURCE_COUNT: 248,
    STATE_CODE: 'TX',
    ZIP: '78748',
  },
  {
    LATITUDE: 42.564518,
    LONGITUDE: -83.068943,
    SOURCE_COUNT: 317,
    STATE_CODE: 'MI',
    ZIP: '48310',
  },
  {
    LATITUDE: 40.395009,
    LONGITUDE: -111.713361,
    SOURCE_COUNT: 1255,
    STATE_CODE: 'UT',
    ZIP: '84062',
  },
  {
    LATITUDE: 37.810102,
    LONGITUDE: -121.91095,
    SOURCE_COUNT: 336,
    STATE_CODE: 'CA',
    ZIP: '94506',
  },
  {
    LATITUDE: 44.276635,
    LONGITUDE: -89.348453,
    SOURCE_COUNT: 36,
    STATE_CODE: 'WI',
    ZIP: '54909',
  },
  {
    LATITUDE: 44.83418,
    LONGITUDE: -91.483057,
    SOURCE_COUNT: 483,
    STATE_CODE: 'WI',
    ZIP: '54703',
  },
];

interface DataProps {
  [key: string]: unknown;
}

export const defaultProcessDataFn = (
  data: Array<DataProps>,
): GeoJSON.FeatureCollection<GeoJSON.Geometry> => {
  return {
    features: data.map(item => {
      return {
        geometry: {
          coordinates: [item.LONGITUDE, item.LATITUDE],
          type: 'Point',
        } as Point,
        properties: {
          quantity: item.SOURCE_COUNT as number,
          stateCode: item.STATE_CODE as string,
          value: item.SOURCE_COUNT as number,
          zip: item.ZIP as string,
        },
        type: 'Feature',
      };
    }),
    type: 'FeatureCollection',
  };
};
