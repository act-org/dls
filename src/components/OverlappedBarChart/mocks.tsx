/**
 * Copyright (c) ACT, Inc. and its affiliates.
 *
 * This source code is licensed under the MIT license found in the
 * LICENSE file in the root directory of this source tree.
 */

export const defaultData = [
  { applicants: 323, inquiries: 6536 },
  { applicants: 312, inquiries: 5550 },
  { applicants: 284, inquiries: 6711 },
];

export const threeBarsData = [
  { applicants: 323, inquiries: 6536, test: 3000 },
  { applicants: 312, inquiries: 5550, test: 3000 },
  { applicants: 284, inquiries: 6711, test: 3000 },
];

export const edgeCaseCloseData = [
  {
    applicants: 9800,
    inquiries: 10255,
  },
  {
    applicants: 10416,
    inquiries: 10416,
  },
  {
    applicants: 11000,
    inquiries: 11418,
  },
];

export const edgeCaseLowData = [
  {
    applicants: 68000,
    inquiries: 313898,
  },
  {
    applicants: 0,
    inquiries: 204352,
  },
  {
    applicants: 9514,
    inquiries: 50000,
  },
];

export const edgeCaseNoData = [
  {
    applicants: 0,
    inquiries: 0,
  },
  {
    applicants: 0,
    inquiries: 0,
  },
  {
    applicants: 0,
    inquiries: 0,
  },
];
