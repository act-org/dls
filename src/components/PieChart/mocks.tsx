/**
 * Copyright (c) ACT, Inc. and its affiliates.
 *
 * This source code is licensed under the MIT license found in the
 * LICENSE file in the root directory of this source tree.
 */

export const defaultData = [
  { name: 'Another Gender', value: 120 },
  { name: 'Male', value: 2200 },
  { name: 'Female', value: 450 },
  { name: 'Not Reported', value: 15 },
];

export const moreOptionsData = [
  {
    description: 'Lorem ipsum dolor sit amet, consectetur adipiscing elit',
    name: 'Experiential Interest',
    value: 220,
  },
  {
    description: 'Quisque tempor ut odio ac accumsan',
    name: 'Career Pragmatics',
    value: 450,
  },
  {
    description: 'Curabitur tempus est tortor, in iaculis mi eleifend non',
    name: 'Social Focus',
    value: 120,
  },
  {
    description:
      'Duis bibendum, lorem sit amet placerat sagittis, ante orci congue nibh, vel maximus ipsum urna eu felis',
    name: 'Exploration & Meaning',
    value: 15,
  },
  {
    description: 'Duis at sodales libero, a aliquet justo',
    name: 'Career Through Academics',
    value: 15,
  },
  { description: 'Sed a tortor urna', name: 'Grad School Bound', value: 15 },
];
