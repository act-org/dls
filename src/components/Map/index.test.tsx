/**
 * Copyright (c) ACT, Inc. and its affiliates.
 *
 * This source code is licensed under the MIT license found in the
 * LICENSE file in the root directory of this source tree.
 *
 * @prettier
 */

import { standard } from '~/helpers/test';

import { defaultData } from './mocks';

import { Map } from '.';

describe('Map', () => {
  const Component = (
    <Map data={defaultData} height={450} mapboxAccessToken="" width="100%" />
  );
  standard(Component);
});
