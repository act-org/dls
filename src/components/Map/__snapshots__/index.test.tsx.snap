// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Map ACT theme matches the snapshot 1`] = `
<div>
  <div
    class="css-1xmgghu"
    style="height: 388px; width: 100%;"
  >
    <div
      style="position: relative; width: 100%; height: 100%;"
    />
  </div>
</div>
`;

exports[`Map ACT_ET theme matches the snapshot 1`] = `
<div>
  <div
    class="css-1xmgghu"
    style="height: 388px; width: 100%;"
  >
    <div
      style="position: relative; width: 100%; height: 100%;"
    />
  </div>
</div>
`;

exports[`Map ENCOURA theme matches the snapshot 1`] = `
<div>
  <div
    class="css-1xmgghu"
    style="height: 388px; width: 100%;"
  >
    <div
      style="position: relative; width: 100%; height: 100%;"
    />
  </div>
</div>
`;

exports[`Map ENCOURA_CLASSIC theme matches the snapshot 1`] = `
<div>
  <div
    class="css-1xmgghu"
    style="height: 388px; width: 100%;"
  >
    <div
      style="position: relative; width: 100%; height: 100%;"
    />
  </div>
</div>
`;

exports[`Map ENCOURAGE theme matches the snapshot 1`] = `
<div>
  <div
    class="css-1xmgghu"
    style="height: 388px; width: 100%;"
  >
    <div
      style="position: relative; width: 100%; height: 100%;"
    />
  </div>
</div>
`;

exports[`Map ENCOURAGE_E4E theme matches the snapshot 1`] = `
<div>
  <div
    class="css-1xmgghu"
    style="height: 388px; width: 100%;"
  >
    <div
      style="position: relative; width: 100%; height: 100%;"
    />
  </div>
</div>
`;
